app/forms/state_file/id_donations_form.rb

```diff
@@ -0,0 +1,30 @@
+module StateFile
+  class IdDonationsForm < QuestionsForm
+    set_attributes_for(
+      :intake,
+      :nongame_wildlife_fund_donation,
+      :childrens_trust_fund_donation,
+      :special_olympics_donation,
+      :guard_reserve_family_donation,
+      :american_red_cross_fund_donation,
+      :veterans_support_fund_donation,
+      :food_bank_fund_donation,
+      :opportunity_scholarship_program_donation
+    )
+
+    validates :nongame_wildlife_fund_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :childrens_trust_fund_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :special_olympics_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :guard_reserve_family_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :american_red_cross_fund_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :veterans_support_fund_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :food_bank_fund_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :opportunity_scholarship_program_donation, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
```

Comment:
[dust] You may consider consolidating these validations. They're all the same, to my eye, so they could be specified on a single line or, if you prefer readability, you may consider capturing them all into an array and then calling something like

```ruby
validates *non_zero_required_fields, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
```

It's a minor matter of taste though tbh